tf_encrypted/keras/layers/convolutional_test.py

```diff
@@ -16,7 +16,10 @@ def setUp(self):
     tf.reset_default_graph()
 
   def test_conv2d_bias(self):
-    self._core_conv2d(kernel_size=2, use_bias=True)
+    self._core_conv2d(kernel_size=2,
+                      use_bias=True,
+                      dilation_rate=(1, 1),
```

Comment:
are new args necessary? thought they were the default

Reply:
they are not, I will remove it. thanks